R/iterate-responses.R

```diff
@@ -0,0 +1,34 @@
+#' Tools for working with lists of responses
+#'
+#' * `resps_combine()` combines the data from each responses into a single
+#'   object.
+#' * `resps_response()` returns all successful responses.
```

Comment:
Maybe `resps_success()` to better match `resps_error()`?

Reply:
I was thinking of these more about as the names of the types you get back, rather success vs failures. Anyway, I'll iterate on these as part of #357.